test-app/package.json

```
@@ -0,0 +1,71 @@
+{
+  "name": "test-app",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Small description for test-app goes here",
+  "repository": "",
+  "license": "MIT",
+  "author": "",
```

```suggestion
  "author": "",
```
As it's private, we can remove it completely, right?